### ISSUE TITLE
negative training loss

### ISSUE THREAD

**Issue by ChuzzZz** [594422814/TransformerTrack#32]
when training trdimp, the loss of bb_ce is negative. Same problem when I tried to train super_dimp. Is the loss of bb_ce supposed to be negative?

**Comment by JackjackFan** [comment]
> when training trdimp, the loss of bb_ce is negative. Same problem when I tried to train super_dimp. Is the loss of bb_ce supposed to be negative?

Hey,have you done this issue? is it normal that got the minus bb_ce loss?
